CHANGELOG.md

```diff
@@ -1256,15 +1256,15 @@
 
 - Only emit m-071, m-072, m-041 if there are fewer than 2 sources
 
-- Add Ukranian to the set of scripts that need a Latn suffix. Thanks to Robin Whittleton
+- Add Ukrainian to the set of scripts that need a Latin suffix. Thanks to Robin Whittleton
```

Comment:
This is incorrect, `Latn` is the actual language keyword used in these attributes

Reply:
This should be fixed by edf1c5e026fddda0ebe438ac8b80d25521d827fc.